vendor/izinto/izi-io-4840-profile-eu868.yaml

```diff
@@ -0,0 +1,24 @@
+# Vendor profile ID, can be freely issued by the vendor
+# This vendor profile ID is also used on the QR code for LoRaWAN devices, see
+# https://lora-alliance.org/sites/default/files/2020-10/LoRa_Alliance_Vendor_ID_for_QR_Code.pdf
+vendorProfileID: 42
```

Comment:
This is probably not right

Reply:
Why? It reads "can be freely issued by the vendor" – and 42 is a nice number.

Reply:
Having read the PDF, this confuses me. Is this (A) a profile ID freely issued by the vendor ( = us, izinto), or (B) a vendor ID issued by LoRa alliance?

Reply:
If you pick 42, that's completely fine, thanks for clarifying.